molecule/openstack/molecule.yml

```diff
@@ -7,7 +7,9 @@ lint:
   options:
     config-file: tests/yamllint.yml
 platforms:
-  - name: test-wipefs
+  - name: test-disk_wipe
+    image: rhel-7.4-server-x86_64-released
```

Comment:
7.4 is old. Let's do this with 7.6, which is current latest, I believe.

Reply:
updated